lib/msal-common/src/client/OnBehalfOfClient.ts

```diff
@@ -121,6 +124,10 @@ export class OnBehalfOfClient extends BaseClient {
         return idTokens[0] as IdTokenEntity;
     }
 
+    private readAccountFromCache(account: AccountInfo): AccountEntity {
+        return this.cacheManager.readAccountFromCache(account);
```

Comment:
Should this be `getAccount`?

Reply:
I think this is a helper used for testing.

Reply:
@sameerag you can probably just mock the cacheManager function instead of mocking this private function if you aren't using it anywhere else.

Reply:
I did. That wasn't working. But this change isn't for that. We want all cache reads through CacheManager and the helper APIs are used across clients.